docs/onduty/onduty.md

```diff
@@ -0,0 +1,54 @@
+# Community On-Duty
+
+Community on-duty is a weekly rotation of Crossplane community members who have
+volunteered to help the Crossplane community. Volunteers are on-duty for a week
+at a time. The goals of the on-duty rotation are to:
+
+1. Ensure the Crossplane community feels heard and supported.
+1. Prevent issues and pull requests (PRs) from falling through the cracks.
+1. Make it easier for the next person on-duty to achieve the above goals.
+
+Volunteers should be able to commit to focus on being on-duty for the majority
+of their typical working day (e.g. ~7 hours a day). On-duty is expected to
+respond to questions, issues and pull requests at their earliest convenience,
+within 24 hours.
+
+> Don't hesitate to reach out to the current person on-duty if you're interested
+> in volunteering but can't commit to so much time - we may be able to work out
+> smaller shifts.
```

Comment:
The commitment above makes sense for the Crossplane maintainers who are paid to work on Crossplane full time, but will likely eliminate basically anyone else from being able to volunteer. We prefer week long shifts because we think it's useful for on-duty to spend their spare time during a week working on improvement projects (e.g. an on-duty dashboard, or improving CI/CD), but day-long shifts would likely be significantly more feasible for folks who aren't paid to work on Crossplane full time.